strings/ctype-latin1.c

```diff
@@ -504,19 +504,22 @@ struct charset_info_st my_charset_latin1_nopad=
  *
  * The modern sort order is used, where:
  *
- * '�'  ->  "ae"
```

Comment:
I strongly think that the readability of this comment is greatly affected by not using the proper latin1 character.

Reply:
To get the code consistent, let's format the files with UTF-8 character set as all the "special characters are in comments". In this case it is more important that the reader understand the meaning, rather than using the "latin-1" encoded gliph. We can use the UTF-8 corresponding byte sequence.

Reply:
@cvicentiu, agreed on this one. The easy readability of the comments is important, since they're discussing the properties of certain characters.

@anson1014 and I had discussed the possibility of converting these comments to UTF-8 instead, using `iconv`, but weren't sure which version would be preferred upstream.

Reply:
These characters have been changed to UTF-8